doc/api/util.md

```diff
@@ -919,6 +988,34 @@ encoded bytes.
 
 The encoding supported by the `TextEncoder` instance. Always set to `'utf-8'`.
 
+## util.triggerNodeReport([filename])
```

Comment:
This accepts an optional `Error` object also, correct?

Reply:
addressed, the whole is modified.